### ISSUE TITLE
[BUG] Bastion terminal connection keeps disconnecting frequently

### ISSUE THREAD

**Issue by dpkshetty** [ocp-power-automation/openshift-install-power#116]
**Describe the bug**
Connection to my bastion terminal window keeps disconnecting time to time

**To Reproduce**
Open a terminal session to Bastion system (using the ssh cmdline in the output post cluster install)
Wait for some time (15-20 mins of no usage)
The terminal is hung.. no echo of any keystroke nor Ctrl-C works!

**Expected behavior**
Stable connection to Bastion host

**Console Output**
```
[root@deepakcs-ocp-psvs-637a-bastion-0 ~]# client_loop: send disconnect: Connection reset by peer
dpkshetty@DESKTOP-L94HV3O:~$
```

**Desktop (please complete the following information):**
 - OS: [e.g. RHEL, Ubuntu]
 - Version [e.g. 8.3, 16.04]
 - Windows terminal [eg: Cygwin, GitBash]

**Additional context**

**Comment by bpradipt** [comment]
@dpkshetty I don't think this is a bug related to the install script. It could be the SSH idle timeout.

**Comment by dpkshetty** [comment]
@bpradipt I don't see this issue when I use OCP on IBM cloud. Since Bastion gets installed as part of the install script, i was hoping there could be some network params we need to fix to avoid this issue ? If not, where would this belong ?

**Comment by bpradipt** [comment]
> @bpradipt I don't see this issue when I use OCP on IBM cloud. Since Bastion gets installed as part of the install script, i was hoping there could be some network params we need to fix to avoid this issue ? If not, where would this belong ?

When using OCP on IBM Cloud were you connecting to bastion? or you were connecting to the OCP nodes directly ?

For bastion, this is an SSH specific setting and ideally this can be done by end user depending on what kind of host specific settings are needed. Likewise for other bastion specific settings which are not directly related to OpenShift usage.

**Comment by dpkshetty** [comment]
> When using OCP on IBM Cloud were you connecting to bastion? or you were connecting to the OCP nodes directly ?

I was usinfg `oc` directly from my local terminal
 
> For bastion, this is an SSH specific setting and ideally this can be done by end user depending on what kind of host specific settings are needed. Likewise for other bastion specific settings which are not directly related to OpenShift usage.

Agree, my point is.. it still leaves a bad taste as far as UX goes :) So if we know what SSH setting needs to be used to avoid the frequent disconnects, better to document it so that the overall OCP on PowerVS experience remains good

**Comment by bpradipt** [comment]
> > When using OCP on IBM Cloud were you connecting to bastion? or you were connecting to the OCP nodes directly ?
> 
> I was usinfg `oc` directly from my local terminal
> 
> > For bastion, this is an SSH specific setting and ideally this can be done by end user depending on what kind of host specific settings are needed. Likewise for other bastion specific settings which are not directly related to OpenShift usage.
> 
> Agree, my point is.. it still leaves a bad taste as far as UX goes :) So if we know what SSH setting needs to be used to avoid the frequent disconnects, better to document it so that the overall OCP on PowerVS experience remains good

I'm happy to document it in FAQs. This will be generic across Power.
